tests/transport/rtps/subscriber.cpp

```diff
@@ -68,17 +67,22 @@ class SimpleDataReader : public TransportReceiveListener, public TransportClient
 
     switch (sample.header_.message_id_) {
     case SAMPLE_DATA: {
-      Serializer ser(sample.sample_.get(),
-                     sample.header_.byte_order_ != ACE_CDR_BYTE_ORDER,
-                     Serializer::ALIGN_CDR);
-      bool ok = true;
+      Serializer ser(sample.sample_.get(), encoding);
+
+      // TODO(iguessthislldo): Convert
       ACE_CDR::ULong encap;
-      ok &= (ser >> encap); // read and ignore 32-bit CDR Encapsulation header
-      TestMsg data;
-      ok &= (ser >> data);
+      if (!(ser >> encap)) {
+        ACE_ERROR((LM_ERROR,
+          "ERROR: data_received() seq# = %d: failed to deserialize encap\n",
+          sample.header_.sequence_.getValue()));
```

Comment:
Looks like a 32-bit/64-bit mismatch with `%d`

Reply:
I just copied it, but yet I can correct them to `%q`.